content/applications/productivity/knowledge/management.rst

```diff
@@ -2,113 +2,240 @@
+Unarchive`.
+
+Version history
+~~~~~~~~~~~~~~~
 
-Articles are organized into a hierarchical structure wherein the article on top is a parent article,
-and those underneath are called nested articles. This structure allows the grouping of related
-articles.
+To restore a previous version of an article, select it in the sidebar tree and click the
+:icon:`fa-history` (:guilabel:`history`) icon in the top-right menu to open the version history.
+Select a version and click :guilabel:`Restore history`.
 
-To establish this hierarchy, create new articles by clicking the :guilabel:`+` button next to the
-parent-to-be article, or move existing articles by either dragging and dropping them under the
-parent-to-be or by clicking the vertical ellipsis button (:guilabel:`⋮`), clicking :guilabel:`Move
-To` on the toolbox and selecting the article to use as a parent.
+.. note::
+   In the version history, the :guilabel:`Content` tab shows the selected version, while the
+   :guilabel:`Comparison` tab displays the differences between an article's previous and current
+   version.
 
-.. _management/categories:
+.. _knowledge/management/access:
 
-Categories
-----------
+Access articles from various apps
+---------------------------------
 
-Additionally, articles are divided into four categories that can be found on the left sidebar. These
-categories give articles default **access rights**.
+Knowledge articles are accessible from the :ref:`form view <studio/views/general/form>` of various
+apps. Click the :icon:`fa-bookmark` :guilabel:`(Knowledge)` icon in the top right corner to open
+the command palette, then perform one of the following methods:
 
-- :guilabel:`Favorites`: You can set any article you can access as a favorite. To do so, click the
-  star-shaped icon (:guilabel:`★`) in the top-right menu of articles. Marking articles as
-  **favorites** is user-specific and does not affect other users.
-- :guilabel:`Workspace`: Articles displayed in that category are available to all internal users.
-  These users have the right to read, modify or share these articles.
-- :guilabel:`Shared`: Articles displayed in that category are those you shared with internal users,
-  external users, or shared with you.
-- :guilabel:`Private`: Articles displayed in that category are only available to you.
+- :guilabel:`Search for an article`: start typing the text to execute a semantic search that
+  identifies relevant article information.
+- :icon:`fa-arrows-alt` :guilabel:`Advanced Search`: after typing the text in the search bar, click
+  :guilabel:`Advanced Search` to perform a parametric search with options to filter, group, or save
+  articles as favorites.
 
-.. image:: management/left-sidebar-cat.png
-   :alt: Categories displayed in the left sidebar of Odoo Knowledge
+.. note::
+  The :icon:`fa-bookmark` :guilabel:`(Knowledge)` icon is only visible if the Knowledge app is
+  installed.
```

Comment:
```suggestion
```
Not really useful as all users reading will either have Knowledge installed or will install it before using this.